packages/oncokb-frontend-commons/src/components/annotation.module.scss

```diff
@@ -0,0 +1,23 @@
+.annotation-item-load {
```

Comment:
This one is also duplicate, but maybe it's okay because it is relatively small?

Reply:
I'm fine with it.